export declare function handleError(err: unknown): void;
